gulpfile.js

```diff
@@ -249,8 +261,7 @@ function buildStrings(out, rootPaths, recursive) {
                             err("cannot JSON-parse " + m[1])
                         }
                     } else {
-                        if (!/util\.ts$/.test(filename))
-                            err("invalid format of lf() argument: " + args)
+                        err("invalid format of lf() argument: " + args)
```

Comment:
// @ignorelf@ needed here?

Reply:
Ah, this one's just part of the gulpfile so we don't pull translations from here / no lf's run in this context -- that said, I'll add it here for consistency with the other spots this function exists.